### DISCUSSION TITLE
Is `spotube.org` fake?

### DISCUSSION THREAD

**Discussion by taivlam** [KRTirtho/spotube#1533] [Q&A]
I came across [`spotube.org`](https://spotube.org/index.html) when trying to search for `spotube` on DuckDuckGo.  I think this is a fake website, as the only official website listed in this GH repo is [`https://spotube.krtirtho.dev/`](https://spotube.krtirtho.dev/).

The logo is copied basically correctly, but the typography for the logo is off.

Also, this site talks about the application as if there's only an Android app (as there are quite a few desktop clients on the official site), and also its downloads page has 1 error: it [links](https://github.com/JunkFood02/Seal/tree/v1.11.0-beta.1) to a release of the Seal app on Android, instead of Spotube; so that seems suspicious.

**Comment by KRTirtho** [comment]
Yup, it's scam website. Currently, it's down though. But, still be aware that only spotube.krtirtho.dev is the original website of Spotube

**Reply by asandikci** [reply]
FYI this website came back
